weather.go

```diff
@@ -1,5 +1,138 @@
 package weather
 
-func Get(location, token string) ([]byte, error) {
-	return nil, nil
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type apiResponse struct {
+	Name    string
+	Weather []struct {
+		Main string
+	}
+	Main struct {
+		Temp float64
+	}
+}
+
+type apiErrorResponse struct {
+	Cod     string
+	Message string
+}
+
+type Conditions struct {
+	City               string
+	OneWord            string
+	TemperatureCelsius float64
+}
+
+func RunCLI(args []string) (output string) {
+
+	token := os.Getenv("WEATHERAPP_TOKEN")
+	if len(token) == 0 {
```

Comment:
It's just slightly clearer (because it's not a function call) to compare `token` with empty string here.

```suggestion
	if token == "" {
```